<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>JobPlannerDialog</class>
 <widget class="QDialog" name="JobPlannerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>656</width>
    <height>581</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Планирование работ</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>4</number>
   </property>
   <property name="margin">
    <number>4</number>
   </property>
   <item>
    <widget class="QSplitter" name="splitter_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="childrenCollapsible">
      <bool>false</bool>
     </property>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="childrenCollapsible">
       <bool>false</bool>
      </property>
      <widget class="CCalendarWidget" name="calendar">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="MinimumExpanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="firstDayOfWeek">
        <enum>Qt::Monday</enum>
       </property>
       <property name="gridVisible">
        <bool>false</bool>
       </property>
      </widget>
      <widget class="QTreeView" name="treeOrgStructure"/>
     </widget>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>0</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="1" column="1">
        <widget class="QPushButton" name="btnPrint">
         <property name="text">
          <string>Печать</string>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QPushButton" name="btnClose">
         <property name="text">
          <string>Закрыть</string>
         </property>
        </widget>
       </item>
       <item row="1" column="2">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>273</width>
           <height>23</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="btnFill">
         <property name="text">
          <string>Заполнить</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="5">
        <widget class="CJobsView" name="tblJobs">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QPushButton" name="btnApply">
         <property name="text">
          <string>Применить</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CCalendarWidget</class>
   <extends>QCalendarWidget</extends>
   <header location="global">library/CalendarWidget.h</header>
  </customwidget>
  <customwidget>
   <class>CJobsView</class>
   <extends>QTableView</extends>
   <header location="global">Resources/Jobs.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>calendar</tabstop>
  <tabstop>treeOrgStructure</tabstop>
  <tabstop>tblJobs</tabstop>
  <tabstop>btnFill</tabstop>
  <tabstop>btnClose</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>btnClose</sender>
   <signal>clicked()</signal>
   <receiver>JobPlannerDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>573</x>
     <y>458</y>
    </hint>
    <hint type="destinationlabel">
     <x>573</x>
     <y>486</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
